breba_app/builder_agent/agent.py

```diff
@@ -258,7 +261,15 @@ async def edit_invoke(self, user_name: str, session_id: str, user_input: Message
             await self.app.ainvoke(Command(update={"current_agent": "editing_spec_agent"}, resume=user_input), config)
         else:
             # This happens in only with the first task request, if task is being continued this will not work
-            logger.info(f"Invoking builder editing agent with user input: {user_input}")
+            snippet = user_input.parts[0].text if user_input.parts else ""
```

Comment:
Again, the full message save a lot of time. Otherwise when I need it, I have to reconstruct the message by hand. Not sure it is always possible.